spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -351,7 +352,8 @@ describe("kong start/stop #" .. strategy, function()
         path = "/hello",
       })
       assert.res_status(404, res) -- no Route configured
-      assert(helpers.stop_kong(helpers.test_conf.prefix))
+      --assert(helpers.stop_kong(helpers.test_conf.prefix))
+      assert(helpers.kong_exec("quit --prefix " .. helpers.test_conf.prefix))
```

Comment:
Perhaps we should change stop_kong using quit.

Likely a bit slower, but it might help with flakyness (less dangling requests).